moveit_ros/visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h

```diff
@@ -46,6 +46,10 @@
 #include <ros/ros.h>
 #endif
 
+#ifdef ROS_KINETIC
+#include <tf2_ros/transform_listener.h>
```

Comment:
This should now be
```suggestion
#include <tf2_ros/buffer.h>
```
as the listener declaration is gone.